setup.py

```diff
@@ -9,7 +9,7 @@
 version = '0'  # set static for now
 
 install_requires = [
-    'acme==0.1.0',
+    'acme==0.4.2',
```

Comment:
Any `0.3` or `0.4` should do: `acme>=0.3,<0.5` (`==` was used previously because there was no clear API on the versioning scheme back then)